Add Math symbols and Malayalam Avagraha (\ and / behaviour)

binilasanki opened the pull request:
Close #16 
Close #17

binilasanki committed:
Add support for consonant, Standardize removal of chandrakkala

  - Aspirated consonants
  - irregular consonants (nga and cha)

binilasanki committed:
Add multiplication symbol

binilasanki committed:
Add division and minus symbols

binilasanki committed:
Add avagraha

binilasanki committed:
Add dot reph

binilasanki committed:
add cha chha sha

binilasanki committed:
add open quotes and closed quotes

binilasanki committed:
add hyphen

binilasanki committed:
add asterisk

binilasanki committed:
add // + / > /

beniza commented on source/mlym-mozhi.kmn:
```diff
@@ -32,10 +32,13 @@ c store(irregularBase) '
 'ച്' + 'h' > 'ഛ്' 
-'സ്' + 'h' > 'ഷ്'  
+'സ്' + 'h' > 'ഷ്' 
+'ൻൿ' + 'h' > 'ഞ്ച്'
```
did you test this code? is it working? please confirm.

beniza reviewed (commented):
I'm not sure if this case will fire as there is a multi letter context in the rule. Could you please check if this code is actually working?Please see my comment also

beniza commented on source/mlym-mozhi.kmn:
```diff
@@ -55,12 +55,19 @@ c malayalam avagraha
 
+c hyphen
+'−' + '\' > '-'  
+
```
too many blank lines....

binilasanki commented:
Yes, it is working

On Tue, Apr 17, 2018 at 10:02 PM, benVar <notifications@github.com> wrote:

> *@beniza* commented on this pull request.
> ------------------------------
>
> In source/mlym-mozhi.kmn
> <https://github.com/beniza/mlym-mozhi-Keyman/pull/22#discussion_r182143967>
> :
>
> >  'ൿ' + 'h' > 'ച്'
>  'ച്' + 'h' > 'ഛ്'
> -'സ്' + 'h' > 'ഷ്'
> +'സ്' + 'h' > 'ഷ്'
> +'ൻൿ' + 'h' > 'ഞ്ച്'
>
> did you test this code? is it working? please confirm.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/beniza/mlym-mozhi-Keyman/pull/22#pullrequestreview-112892391>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AkRHQu8Oix4e-LGOR313FlC5YeBpQajnks5tphkMgaJpZM4TMalt>
> .
>

binilasanki committed:
Complete Basic Malayalam Characters

Add Malayalam Vowel Matra
Add Anusvara (as part of extended chillu)
Add Malayalm fluid (ya-ha)

binilasanki committed:
Extend suport for irregular forms

Add irregular Matra handling (ka + i = kai)
Add RR
Create superstore for Malayalam Consonants

binilasanki committed:
More irregular cases

binilasanki committed:
add ma virama

binilasanki commented:
any(chilluExtendedBase)  + any(fluidsMlym) > index(chilluExtendedBase, 1)
'്' index(fluidsMlym, 1)
Is it correct for  ന്ര ?

On Thu, May 3, 2018 at 12:22 PM, benVar <notifications@github.com> wrote:

> Merged #22 <https://github.com/beniza/mlym-mozhi-Keyman/pull/22>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/beniza/mlym-mozhi-Keyman/pull/22#event-1607001058>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AkRHQo5as1Kh5YhLew5N3fQepsL09Yzbks5tuqkbgaJpZM4TMalt>
> .
>
